app/src/lib/feature-flag.ts

```diff
@@ -39,5 +39,5 @@ export function enableNotificationOfBranchUpdates(): boolean {
 
 /** Should the repository list display info indicators? */
 export function enableRepoInfoIndicators(): boolean {
-  return enableDevelopmentFeatures()
+  return enableBetaFeatures()
```

Comment:
Should this correspond with a 'new' for repo list info indicators?

Reply:
It's currently in the list as an `Improved` entry:

```
[Improved] Repository list now displays uncommitted changes count and ahead/behind information - #2259
```

I don't have any strong feels on whether we want to call it out with `New` or save that for bigger-bang features. Thoughts @desktop/core @desktop/comrades?

Reply:
I'd definitely consider this to be a **new** entry